rm_reporting/controllers/case_controller.py

```diff
@@ -25,8 +26,13 @@ def get_case_data(collection_exercise_id: str) -> list:
         "ORDER BY sample_unit_ref, status"
     )
 
-    with case_engine.begin() as conn:
-        return conn.execute(case_business_ids_query, {"collection_exercise_id": collection_exercise_id}).all()
+    try:
+        with case_engine.begin() as conn:
+            result = conn.execute(case_business_ids_query, {"collection_exercise_id": collection_exercise_id}).all()
+    except SQLAlchemyError:
+        logger.error("Failed to get case data", collection_exercise_id=collection_exercise_id)
```

Comment:
In this log we need to include the `e.__class__.__name__` in the log so we know what specific error is being raised

Reply:
Done.